.travis.yml

```diff
@@ -8,6 +8,7 @@ node_js:
 addons:
   apt:
     packages:
+	  - libudev-dev
```

Comment:
Unfortunately, YML only supports space indentation (I think 2-space specifically).  I believe having a tab in here will cause problems when this file is read.